CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
 # Contributing to Real-time Ubuntu
+
+The [Real-time Ubuntu documentation] lives in the `docs/` directory and is deployed on Read the Docs.
```

Comment:
...lives in the `docs/` directory of this repository; you'll need a GitHub account to make contributions.

Reply:
Isn't that obvious, since the instructions and the source to be edited is on Github?

Reply:
Hi @farshidtz , please review the newly-added suggestions only. The rest of the previous comments are no longer relevant.

Thanks.